recipes/oddt/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "oddt" %}
+{% set version = "0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 330f510a6563c371627370200fbfb39ed3496662d067d17e09edb5a6025c8ae6
+
+build:
+  skip: true  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - six
+    - numpy >=1.11
```

Comment:
Since you are using numpy, please follow the instructions here, https://conda-forge.readthedocs.io/en/latest/meta.html#building-against-numpy.

Reply:
Numpy is used nut not linked. Should I still pin it?

Reply:
Yes, please.